python/core/meet-python/why-learn-python.md

```diff
@@ -32,7 +33,17 @@ for i in items:
 
 Python is also an economical programming language. For similar tasks, Python can often require *3-5* times less code than Java, and up to *10* times less code than C++.
 
-Python is an *object-oriented* programming language, therefore it can act as a stepping stone to learning new object-oriented languages, and can help you adapt to other programming environments.
+As we know, Python is an *object oriented* programming language, therefore because of Python's object oriented nature it can act as a stepping stone to learning new object oriented languages, and can help you adapt to other programming environments. Nonetheless, Python is also used as functional programming language in certain conditions. 
+
+---
+## Practice
+
+What is the *pythonic* way to write a for loop? 
```

Comment:
Expand the code sample to include an actual loop
How does this discuss what's covered in the article?

Reply:
@lizTheDeveloper please review this question again, I tried to change it such that it reflects the objectives of this insight, not sure whether it is accurate tho.